programs/clearing_house/src/controller/validate.rs

```diff
@@ -224,3 +225,40 @@ pub fn validate_position_account(
 
     Ok(())
 }
+
+pub fn validate_spot_position(position: &SpotPosition) -> ClearingHouseResult {
```

Comment:
as a separate task would be nice to move these to the `validation` folder

Reply:
done. good call